README.md

```diff
@@ -23,3 +27,224 @@ List of apps and services used:
 5. **Discord**: chatting and communication.
 6. **Figma**: prototyping and designing UI/UX.
 7. **VSCode**: development and writing code.
+
+# UI/UX
+
+- Ideas to talk about
+    - Why UI/UX is important
+    - Choosing the design system and components library
+    - Why do you build components in Figma?
+    - The thinking process while designing a page
+        - افهم المطلوب
+        - اكتب تفاصيل كل صفحة وأقسامها
+        - شخبط على ورق **وانظر لأمثلة وإلهامات**
+        - نفذ وبطل أفورة
+    
+
+# Frontend architecture
+
+- Didn’t Documented Yet
+    - Testing ⇒ Cypress & Jest & React Query Axios
+    - SEO
+    - Accessibility
+    - Performance
+
```

Comment:
I think you should focus on the backend documentation only, and leave other sections at least empty!